compiler+runtime/src/cpp/jank/codegen/llvm_processor.cpp

```diff
@@ -1669,17 +1714,294 @@ namespace jank::codegen
     return ret;
   }
 
+  void llvm_processor::impl::route_unhandled_exception(llvm::Value *ex_ptr,
```

Comment:
Starting here, we have nearly 300 lines of code which are not documented. There are a few comments, but they all say WHAT, not WHY. We need to do more here.